java/client/src/main/java/glide/api/commands/SortedSetCommands.java

```diff
@@ -0,0 +1,87 @@
+ *
+ * @see <a href="https://redis.io/commands/?group=sorted-set">Sorted set Commands</a>
+ */
+public interface SortedSetCommands {
+    /**
+     * Adds members with their scores to the sorted set stored at `key`. If a member is already a part
+     * of the sorted set, its score is updated.
+     *
+     * @see <a href="https://redis.io/commands/zadd/">redis.io</a> for more details.
+     * @param key - The key of the sorted set.
+     * @param membersScoresMap - A mapping of members to their corresponding scores.
+     * @param options - The Zadd options.
+     * @param changed - Modify the return value from the number of new elements added, to the total
+     *     number of elements changed.
+     * @returns The number of elements added to the sorted set. If `changed` is set, returns the
+     *     number of elements updated in the sorted set. If `key` holds a value that is not a sorted
+     *     set, an error is returned.
+     * @example
+     *     <p><code>
+     * client.zadd("mySortedSet", Map.of("member1", 10.5d, "member2", 8.2d), ZaddOptions.builder().build(), false).get();
+     * 2 (Indicates that two elements have been added or updated in the sorted set "mySortedSet".)
+     *
+     * client.zadd("existingSortedSet", Map.of("member1", 15.0d, "member2", 5.5d), ZaddOptions.builder().conditionalChange(ZaddOptions.ConditionalChange.ONLY_IF_EXISTS).build(), false).get();
+     * 2 (Updates the scores of two existing members in the sorted set "existingSortedSet".)
+     * </code>
+     */
+    CompletableFuture<Long> zadd(
+            String key, Map<String, Double> membersScoresMap, ZaddOptions options, boolean changed);
+
+    /** Increments the score of member in the sorted set stored at `key` by `increment`.
+     * If `member` does not exist in the sorted set, it is added with `increment` as its score (as if its previous score was 0.0).
+     * If `key` does not exist, a new sorted set with the specified member as its sole member is created.
+     *
+     * @see <a href="https://redis.io/commands/zadd/">redis.io</a> for more details.
+     * @param key - The key of the sorted set.
+     * @param member - A member in the sorted set to increment.
+     * @param increment - The score to increment the member.
+     * @param options - The Zadd options.
+     * @returns The score of the member.
+     * If there was a conflict with the options, the operation aborts and null is returned.
+     * If `key` holds a value that is not a sorted set, an error is returned.
+     *
+     * @example
+     * <p><code>
+     * client.zaddIncr("mySortedSet", member, 5.0d, ZaddOptions.builder().build()).get();
+     * 5.0
+     *
+     * client.zaddIncr("existingSortedSet", member, 3.0d, ZaddOptions.builder().updateOptions(ZaddOptions.UpdateOptions.SCORE_LESS_THAN_CURRENT).build()).get();
+     * null
+     */
+    CompletableFuture<Double> zaddIncr(
+            String key, String member, double increment, ZaddOptions options);
```

Comment:
Please add signature for when no ZaddOptions are provided:
```
CompletableFuture<Double> zaddIncr(
            String key, String member, double increment);
```

Reply:
Added